### ISSUE TITLE
Addon manager and pypi packages: any intentions to put them together?

### ISSUE THREAD

**Issue by joha2** [FreeCAD/FreeCAD-addons#71]
Heyhey,

short time ago I started a thread in the forum about refactoring an addon (pyrate) such that it is usable with PyPI. The answers were very helpful, but looo told me that this refactoring would break the usability of the package as an addon. However it is possible to install the package by PyPI, rendering a pre requisite checker obsolete, because it is done by PyPI. Is there anything planned in the future to integrate PyPI into the addon manager to make it possible to use extensions from outside of FreeCAD?

**Comment by looooo** [comment]
Yes definitely. It would be easy to do so. I guess the easiest option is to write a totally new tool and integrate it into the addon-manager as another tab. I guess all we need is a file which contains all the pip installable freecad-modules.

**Comment by luzpaz** [comment]
I'd suggest re-starting the above [thread](https://forum.freecadweb.org/viewtopic.php?f=22&t=27396) on the forum. Maybe even wireframing what the added tab in the Addon Manager would look like ?

**Comment by looooo** [comment]
yes I will do. I think most of the things are done. https://github.com/looooo/freecad_pipintegration

**Comment by looooo** [comment]
@luzpaz thanks, done https://forum.freecad.org/viewtopic.php?f=22&t=29584

**Comment by PhoneDroid** [comment]
@looooo Is this still relevant?

**Comment by looooo** [comment]
I didnt continue with this, but still think tradional python packaging for addons, workbenches would be a big benefit for freecad.

**Comment by PhoneDroid** [comment]
@looooo Currently not realistic, we are having trouble 
to get people to even use git to serve their addons.
